lib/active_fedora/change_set.rb

```diff
@@ -19,8 +19,8 @@ def empty?
 
     def changes
       @changes ||= changed_attributes.each_with_object({}) do |key, result|
-        if m = /^.*_ids?$/.match(key)
-          predicate = object.association(m[0].to_sym).reflection.predicate
+        if object.respond_to?(:association) && object.association(key.to_sym).present?
```

Comment:
why are you testing to see if object `responds_to?(:association)`? Isn't that always true?